packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -273,6 +290,19 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
     return oracles[token];
   }
 
+  /**
+   * @notice Returns the expiry for the token if exists, if not the default
+   * @param token The address of the token
+   * @return The report expiry in seconds
+   */
+  function getTokenReportExpirySeconds(address token) public view returns (uint256) {
+    if (tokenReportExpirySeconds[token] == 0) {
+      return reportExpirySeconds;
+    }
+
+    return tokenReportExpirySeconds[token];
+  }
```

Comment:
nit: Do we usually write tests for this kind of functions too?

Reply:
Yes you're right, I initially had that as internal so I didn't test it, but then decided to expose it to contractkit so I should add tests :+1: